tinker/e_announcements/__init__.py

```diff
@@ -27,10 +26,12 @@ def __init__(self):
     def before_request(self, name, **kwargs):
         pass
 
+    @cache.memoize(timeout=600)
```

Comment:
This will force everyone's e-annz homepage to be the same - instead of having their specific e-annz